templates/base.html

```diff
@@ -0,0 +1,34 @@
+<html lang="en">
+    <head>

```

Comment:
sorry that I didn't notice it before) but
using 2 whitespaces is the best practice
https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD
please, change everywhere